internal/server/models/settings.go

```diff
@@ -31,7 +31,7 @@ func (s *Settings) ToAPI() *api.Settings {
 }
 
 func (s *Settings) SetFromAPI(a *api.Settings) {
```

Comment:
We should open an issue for this and add some API validators.

Reply:
It's actually not as I originally thought. The only validation that's critical is to set a minimum minimum length since the default value of 0 is not valid. 8 is probably a good starting point